pkg/services/ngalert/schedule/schedule.go

```diff
@@ -314,8 +323,12 @@ func (sch *schedule) ruleEvaluationLoop(ctx context.Context) error {
 		select {
 		case tick := <-sch.heartbeat.C:
 			tickNum := tick.Unix() / int64(sch.baseInterval.Seconds())
-			alertRules := sch.fetchAllDetails()
-			sch.log.Debug("alert rules fetched", "count", len(alertRules))
+			disabledOrgsArray := make([]int64, 0, len(sch.disabledOrgs))
```

Comment:
I think we can just call this `disabledOrgs`?

Reply:
yes, I just wanted to highlight that this is an array as opposed to `sch.disabledOrgs` which is a map.